flexmeasures/cli/data_add.py

```diff
@@ -1871,6 +1880,72 @@ def create_power_asset(
             f"The sensor recording the power of the shiftable load is {shiftable_power} (ID: {shiftable_power.id}).",
             **MsgStyle.SUCCESS,
         )
+    elif kind == "reporter":
+        # Part A) of tutorial IV
+        grid_connection_capacity = get_or_create_model(
+            Sensor,
+            name="grid connection capacity",
+            generic_asset=nl_zone,
+            timezone="Europe/Amsterdam",
+            event_resolution="P1Y",
+            unit="MW",
+        )
+        db.session.commit()
+
+        click.secho(
+            f"The sensor storing the grid connection capacity of the building is {grid_connection_capacity} (ID: {grid_connection_capacity.id}).",
+            **MsgStyle.SUCCESS,
+        )
+
+        tz = pytz.timezone(app.config.get("FLEXMEASURES_TIMEZONE", "Europe/Amsterdam"))
+        current_year = datetime.now().year
+        start_year = datetime(current_year, 1, 1)
+
+        belief = TimedBelief(
+            event_start=tz.localize(start_year),
+            belief_time=tz.localize(datetime.now()),
+            event_value=0.5,
+            source=DataSource.query.get(1),
+            sensor=grid_connection_capacity,
+        )
+
+        db.session.add(belief)
+        db.session.commit()
+
+        headroom = create_asset_with_one_sensor(
+            "toy-battery",
+            "battery",
+            "headroom",
+        )
+
+        db.session.commit()
+
+        click.secho(
+            f"The sensor storing the headroom is {headroom} (ID: {headroom.id}).",
+            **MsgStyle.SUCCESS,
+        )
+
+        for name in ["Inflexible", "Breakable", "Shiftable"]:
+            loss_sensor = create_asset_with_one_sensor(
+                "toy-process", "process", f"Losses ({name})", unit="EUR"
```

Comment:
From https://github.com/FlexMeasures/screenshots/pull/4#pullrequestreview-1587357614: I propose to name the sensor `f"costs ({name})"`, to make the example click.